src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1112,6 +1093,7 @@ public int hashCode() {
      * @return a boolean indicating whether the collection has changed or not.
      * @throws NullPointerException if the collection or iterator is null
      */
+    @SuppressWarnings("unchecked")
```

Comment:
Hmm, It is very interesting. My IDE, (i use intellij idea), warns me about _unchecked cast_. Maybe,  it is a false warning. if you think that is not necessary, i will not persist :)